### PR TITLE
Add desc about initiateChatOnLoad config

### PR DESCRIPTION
**Pull request by rehanHS**

### CODE REVIEW

File: docs/web-chat/configuration.mdx [hunk 1]
```diff
@@ -138,9 +138,9 @@ If you are creating issues with the `fullPrivacy` option set to `true`, then don
 
 </Admonition>
 
-###Example Embed Code
+#### Example Embed Code
```
Comment (hs-saunved) [inline docs/web-chat/configuration.mdx:141]:
```suggestion
### Example Embed Code
```

File: docs/web-chat/configuration.mdx [hunk 2]
```diff
@@ -151,3 +151,20 @@ window.helpshiftConfig = {
   fullPrivacy: true
 };
 ```
+
+## Initiating chat {#initiate-chat}
+
+Developers can start a new chat when the previous issue is resolved, without needing any user action like clicking the `New Conversation` button or going through the post resolution flow like feedback bot for the previously resolved issue.
+
+To do the above mentioned action developer will have to set `initiateChatOnLoad` config in `window.helpshiftConfig` and then use the [updateHelpshiftConfig](https://developers.helpshift.com/web-chat/api/#update-helpshift-config) api to start the new chat when the previous issue is resolved.
```
Comment (hs-saunved) [inline docs/web-chat/configuration.mdx:159]:
```suggestion
To do the above mentioned action, the developer will have to set `initiateChatOnLoad` config in `window.helpshiftConfig` and then use the [updateHelpshiftConfig](https://developers.helpshift.com/web-chat/api/#update-helpshift-config) API to start the new chat when the previous issue is resolved.
```

File: docs/web-chat/configuration.mdx [hunk 3]
```diff
@@ -151,3 +151,20 @@ window.helpshiftConfig = {
...
+#### Example Embed Code
+
+```javascript
+
+window.helpshiftConfig.initiateChatOnLoad = true;
+
+Helpshift("updateHelpshiftConfig");
+
+```
+To open the widget [open](https://developers.helpshift.com/web-chat/api/#open-widget) api can be used.
```
Comment (hs-saunved) [inline docs/web-chat/configuration.mdx:170]:
```suggestion
To open the widget [open](https://developers.helpshift.com/web-chat/api/#open-widget) API can be used.
```

**Review by hs-saunved — CHANGES_REQUESTED** [review]

File: docs/web-chat/configuration.mdx (hunk 1 above)
Comment (rehanHS) [reply docs/web-chat/configuration.mdx:141]: As discussed making it 5 hashes

File: docs/web-chat/configuration.mdx (hunk 2 above)
Comment (rehanHS) [reply docs/web-chat/configuration.mdx:159]: Ack

File: docs/web-chat/configuration.mdx (hunk 3 above)
Comment (rehanHS) [reply docs/web-chat/configuration.mdx:170]: Ack

**Review by helpshift-vishal — APPROVED** [review]
Approved

**Review by hs-hritik — APPROVED** [review]

**Pull request merged**